test/core/end2end/fake_resolver.c

```diff
@@ -74,6 +79,39 @@ typedef struct {
   grpc_channel_args** target_result;
 } fake_resolver;
 
+grpc_arg grpc_fake_resolver_lb_enabled_arg() {
+  grpc_arg arg;
+  arg.type = GRPC_ARG_INTEGER;
+  arg.key = GRPC_ARG_FAKE_RESOLVER_LB_ENABLED;
+  arg.value.integer = 1;
+  return arg;
+}
+
+grpc_arg grpc_fake_resolver_balancer_names_arg(
+    const char* comma_separated_balancer_names) {
+  grpc_arg arg;
+  arg.type = GRPC_ARG_STRING;
+  arg.key = GRPC_ARG_FAKE_RESOLVER_BALANCER_NAMES;
+  arg.value.string = gpr_strdup(comma_separated_balancer_names);
+  return arg;
+}
+
+bool grpc_fake_resolver_get_lb_enabled(const grpc_channel_args* args) {
+  const grpc_arg* arg =
+      grpc_channel_args_find(args, GRPC_ARG_FAKE_RESOLVER_LB_ENABLED);
+  if (arg == NULL) return false;
+  const grpc_integer_options opts = {0, 0, INT_MAX};
```

Comment:
Added the suggested function. However, in doing so I've used `INT_MAX` again (in addition to `INT_MIN`) because any int value can be evaluated as a bool.